modules/minimum-ibm-z-system-requirements.adoc

```diff
@@ -12,6 +27,12 @@ You can install {product-title} version {product-version} on the following {ibm-
 * {ibm-name} z16 (all models), {ibm-name} z15 (all models), {ibm-name} z14 (all models)
 * {ibm-linuxone-name} 4 (all models), {ibm-linuxone-name} III (all models), {ibm-linuxone-name} Emperor II, {ibm-linuxone-name} Rockhopper II
 
+[WARNING]
+====
+Running {product-title} on {ibm-z-name} in processor resource/system manager (PR/SM) mode without a hypervisor is strongly discouraged. Instead, use the Dynamic Partition Manager (DPM).
```

Comment:
Generally we should avoid recommendations, or in this case discouraging [1], instead explaining why something should or should not be down. I suspect that the KB you are waiting on might explain why. I suggest rewording and letting the KB do the heavy lifting here.

[1] https://redhat-documentation.github.io/supplementary-style-guide/#recommend

Reply:
Correct wording is under discussion. I'll change once we have an agreement.